ethcore/types/src/account_diff.rs

```diff
@@ -16,12 +16,9 @@
 
 //! Diff between two accounts.
 
-use std::cmp::*;
-use std::fmt;
 use std::collections::BTreeMap;
-use std::convert::TryFrom;
-use ethereum_types::{BigEndianHash as _, H256, U256};
```

Comment:
BigEndianHash is used [here](https://github.com/paritytech/parity-ethereum/blob/a729ca3223f44773ba27a1a5f87f87a73dbefd89/ethcore/types/src/account_diff.rs#L108)

Reply:
this code has been removed as well